lib/engines/bh.js

```diff
@@ -47,7 +46,7 @@ module.exports = function (opts) {
 
         // drop cache
         if (opts.force || options.forceExec || options.forceLoad) {
-            dropRequireCache(require, filePath);
+            require('enb/lib/fs/drop-require-cache')(require, filePath);
```

Comment:
That's fine, but we can also store the result of `require(...)` to `dropRequireCache`

Like:
```
dropRequireCache = dropRequireCache || require(...);
dropRequireCache(require, filePath);
```